packages/core/docs/commercetools/use-facet.md

```diff
@@ -186,7 +186,7 @@ If the explicit configuration is not provided, the following defaults will be us
   ],
   subcategoriesLimit: 100,
   availableFacets: [
-    { facet: 'categories.id', type: 'string', option: 'subtree("*")', name: 'category' },
+    { facet: 'categories.id', type: 'string', option: 'subtree("*")', name: 'category' }, // Don't change the "name" of this facet
```

Comment:
Linter yields all the time on me to have comments - line before the line. Let's apply it there too

Reply:
This comment is used to grab users' attention, not to be kept in code. I think that having it on the same line as the code it refers to makes it slightly better in this case.